### PR TITLE
Add project tenets

### PR DESCRIPTION
**Pull request by TerryHowe**
Tenets for the project

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/contents/index.md [hunk 1]
```diff
@@ -34,6 +34,18 @@ refer to the [Build](users/build.md) instructions to build a cluster from scratc
 See the [Partners](community/partners.md) page for links to third-party methods for
 installing EKS-D.
 
+## Project Tenets
+
+The tenets of the EKS Distro (EKS-D) project are:
+
+1. **The Source**: The goal of the EKS Distro is to be the Kubernetes source for EKS and EKS Anywhere
+2. **Simple**: make Kubernetes operations, administration, and management simple and boring
```
Comment (micahhausler) [inline docs/contents/index.md:42]:
```suggestion
2. **Simple**: Make Kubernetes operations, administration, and management simple and boring
```

File: docs/contents/index.md [hunk 2]
```diff
@@ -34,6 +34,18 @@ refer to the [Build](users/build.md) instructions to build a cluster from scratc
...
+3. **Opinionated Modularity**: provide opinionated defaults about the best tooling to include with Kubernetes, but give customers the ability to swap them out
```
Comment (micahhausler) [inline docs/contents/index.md:43]:
```suggestion
3. **Opinionated Modularity**: Provide opinionated defaults about the best tooling to include with Kubernetes, but give customers the ability to swap them out
```

File: docs/contents/index.md [hunk 3]
```diff
@@ -34,6 +34,18 @@ refer to the [Build](users/build.md) instructions to build a cluster from scratc
...
+4. **Open**: provide open source tooling backed, validated and supported by Amazon
```
Comment (micahhausler) [inline docs/contents/index.md:44]:
```suggestion
4. **Open**: Provide open source tooling backed, validated, and maintained by Amazon
```

File: docs/contents/index.md [hunk 4]
```diff
@@ -34,6 +34,18 @@ refer to the [Build](users/build.md) instructions to build a cluster from scratc
...
+5. **Ubiquitous**: provide support for the most common public cloud, on-premises bare metal, and on-premises virtualized deployment targets
+6. **Stand Alone**: provided for use anywhere without AWS dependencies (on-premises air-gapped)
```
Comment (micahhausler) [inline docs/contents/index.md:46]:
```suggestion
6. **Stand Alone**: Provided for use anywhere without AWS dependencies (on-premises air-gapped)
```

File: docs/contents/index.md [hunk 5]
```diff
@@ -34,6 +34,18 @@ refer to the [Build](users/build.md) instructions to build a cluster from scratc
...
 
+## Project Tenets
+
+The tenets of the EKS Distro (EKS-D) project are:
+
+1. **The Source**: The goal of the EKS Distro is to be the Kubernetes source for EKS and EKS Anywhere
+2. **Simple**: make Kubernetes operations, administration, and management simple and boring
+3. **Opinionated Modularity**: provide opinionated defaults about the best tooling to include with Kubernetes, but give customers the ability to swap them out
+4. **Open**: provide open source tooling backed, validated and supported by Amazon
+5. **Ubiquitous**: provide support for the most common public cloud, on-premises bare metal, and on-premises virtualized deployment targets
```
Comment (micahhausler) [inline docs/contents/index.md:45]:
```suggestion
5. **Ubiquitous**: Enable customers to run Kubernetes on the most common public cloud, on-premises bare metal, and on-premises virtualized deployment targets
```

File: docs/contents/index.md [hunk 6]
```diff
@@ -34,6 +34,18 @@ refer to the [Build](users/build.md) instructions to build a cluster from scratc
...
+6. **Stand Alone**: provided for use anywhere without AWS dependencies (on-premises air-gapped)
+7. **Better with AWS**: enable AWS customers to enlist AWS Enterprise Support and allow them easily adopt additional AWS services
```
Comment (micahhausler) [inline docs/contents/index.md:47]: I like the title, lets change the wording
```suggestion
7. **Better with AWS**: Enable AWS customers to easily adopt additional AWS services
8. ```

**Review by micahhausler — CHANGES_REQUESTED** [review]
Remove most of the support wording

**Review by micahhausler — APPROVED** [review]
/hold

feel free to unhold if there are no more changes

/lgtm
/approve

File: docs/contents/index.md [hunk 7]
```diff
@@ -34,6 +34,18 @@ refer to the [Build](users/build.md) instructions to build a cluster from scratc
 See the [Partners](community/partners.md) page for links to third-party methods for
 installing EKS-D.
 
+## Project Tenets
+
+The tenets of the EKS Distro (EKS-D) project are:
+
+1. **The Source**: The goal of the EKS Distro is to be the Kubernetes source for EKS and EKS Anywhere
+2. **Simple**: Make Kubernetes operations, administration, and management simple and boring
```
Comment (sostheim) [inline docs/contents/index.md:42]: Focus on EKS-D tenets here:
> Simple: Make using a Kubernetes distribution simple and boring (reliable and secure).

File: docs/contents/index.md [hunk 8]
```diff
@@ -34,6 +34,18 @@ refer to the [Build](users/build.md) instructions to build a cluster from scratc
...
+3. **Opinionated Modularity**: Provide opinionated defaults about the best tooling to include with Kubernetes, but give customers the ability to swap them out
```
Comment (sostheim) [inline docs/contents/index.md:43]: Again, more eks-d focused: s/tooling/components/.

File: docs/contents/index.md [hunk 9]
```diff
@@ -34,6 +34,18 @@ refer to the [Build](users/build.md) instructions to build a cluster from scratc
...
+4. **Open**: provide open source tooling backed, validated and maintained by Amazon
+5. **Ubiquitous**: Enable customers to run Kubernetes on the most common public cloud, on-premises bare metal, and on-premises virtualized deployment targets
```
Comment (sostheim) [inline docs/contents/index.md:45]: EKS-D focused: 
> Ubiquitous: Enable customers and partners to integrate a Kubernetes distribution in the most common tooling (Kubernetes installers and distributions, infrastructure as code, and more).

File: docs/contents/index.md [hunk 10]
```diff
@@ -34,6 +34,18 @@ refer to the [Build](users/build.md) instructions to build a cluster from scratc
...
+6. **Stand Alone**: Provided for use anywhere without AWS dependencies (on-premises air-gapped)
```
Comment (sostheim) [inline docs/contents/index.md:46]: drop on-prem and air gapped distinction here.

File: docs/contents/index.md [hunk 11]
```diff
@@ -34,6 +34,18 @@ refer to the [Build](users/build.md) instructions to build a cluster from scratc
...
+7. **Better with AWS**: Enable AWS customers to easily adopt additional AWS services
```
Comment (sostheim) [inline docs/contents/index.md:47]: for the eks-d focused tentets - I think this is not relevant.

**Review by sostheim — CHANGES_REQUESTED** [review]

File: docs/contents/index.md (hunk 11 above)
Comment (micahhausler) [reply docs/contents/index.md:47]: Lets keep it, we do include AWS-specific components (authenticator) and could include the KMS encryption provider, cloud provider, or others in the future

File: docs/contents/index.md [hunk 12]
```diff
@@ -34,6 +34,18 @@ refer to the [Build](users/build.md) instructions to build a cluster from scratc
...
 installing EKS-D.
 
+## Project Tenets
+
+The tenets of the EKS Distro (EKS-D) project are:
+
+1. **The Source**: The goal of the EKS Distro is to be the Kubernetes source for EKS and EKS Anywhere
+2. **Simple**: Make Kubernetes operations, administration, and management simple and boring
+3. **Opinionated Modularity**: Provide opinionated defaults about the best tooling to include with Kubernetes, but give customers the ability to swap them out
+4. **Open**: provide open source tooling backed, validated and maintained by Amazon
```
Comment (micahhausler) [inline docs/contents/index.md:44]: nit
```suggestion
4. **Open**: Provide open source tooling backed, validated and maintained by Amazon
```

File: docs/contents/index.md (hunk 6 above)
Comment (sostheim) [reply docs/contents/index.md]: lgtm

File: docs/contents/index.md (hunk 11 above)
Comment (sostheim) [reply docs/contents/index.md:47]: sgtm

**Review by sostheim — APPROVED** [review]

**Review by micahhausler — APPROVED** [review]
/lgtm

**Review by micahhausler — APPROVED** [review]
/unhold
/lgtm

**Pull request merged**